week0/homework/Person.js

```diff
@@ -0,0 +1,20 @@
+module.exports =  function (name,age)  {
+   let arr = [];
+   this.name = name;
+   this.age = age;
+   this.addPhone = function(phone) {
+        arr.push({"Name":this.name,"Age":this.age,"Phone":phone});
+   }
+
+  this.call = function() {
+       var phone = arr.map(function(o){return o.Phone});
+       if (phone != "") {
+           return "Calling " + this.name + " at " + phone + " ...";
+       } else {
+           return this.name + " has no phone number saved.";
+       }
+   }
+   this.birthday = function() {
+       return "Wishing " + this.name + " a happy 29th birthday!";
```

Comment:
The point here was to increment and use the age property, and not to hard-code the display. Of course it wasn't mentioned explicitly in the example, but you should always remain skeptical of hard-coding unless you see it absolutely necessary in some way.